app/src/main/java/rocks/mobileera/mobileera/utils/Favorites.kt

```diff
@@ -0,0 +1,13 @@
+package rocks.mobileera.mobileera.utils
+
+import android.content.Context
+import android.support.annotation.DrawableRes
+import rocks.mobileera.mobileera.R
+import rocks.mobileera.mobileera.model.Session
+
+@DrawableRes
+fun favoriteIconResForSession(session: Session?, context: Context): Int {
```

Comment:
I would prefer passing in a non-null `Session` here, but that's a larger refactor that I didn't wanna dive into for this simple fix.